xbout/load.py

```diff
@@ -243,6 +255,10 @@ def collect(varname, xind=None, yind=None, zind=None, tind=None,
         # If BOUT Version is not saved in the dataset
         version = 0
 
+    # Collect just stores iteration from BOUT.dmp.0 as a scalar
+    latest_top_left = {dim: 0 for dim in ds['iteration'].dims}
+    ds['iteration'] = ds['iteration'].isel(latest_top_left).squeeze(drop=True)
```

Comment:
Is this needed if `iteration` is handled in `open_boutdataset`?

Reply:
No, you're right